src/lightning/fabric/utilities/seed.py

```diff
@@ -110,7 +110,7 @@ def _collect_rng_states(include_cuda: bool = True) -> Dict[str, Any]:
         "python": python_get_rng_state(),
     }
     if include_cuda:
-        states["torch.cuda"] = torch.cuda.get_rng_state_all()
+        states["torch.cuda"] = torch.cuda.get_rng_state_all() if torch.cuda.is_available() else []
```

Comment:
```suggestion
    if include_cuda and torch.cuda.is_available():
        states["torch.cuda"] = torch.cuda.get_rng_state_all()
```

Is there a reason why you create the key with an empty list?


`_set_rng_states` checks if the key exists

Reply:
I find that more practical given the `include_cuda` setting.